src/connectors/infinite-hits/__tests__/connectInfiniteHits-test.ts

```diff
@@ -435,14 +439,12 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/infinite-hi
     const makeWidget = connectInfiniteHits(renderFn);
     const widget = makeWidget({
       transformItems: items =>
-        items.map(item => ({
-          ...item,
-          _highlightResult: {
-            name: {
-              value: item._highlightResult.name.value.toUpperCase(),
-            },
-          },
-        })),
+        items.map(item => {
+          // @ts-ignore
+          item._highlightResult.name.value = item._highlightResult.name.value.toUpperCase();
```

Comment:
Someone knows where the name is coming from?

Reply:
I don't understand your question, but from here, it's from  here,  no? https://github.com/algolia/instantsearch.js/blob/d5f21a4e9a0230e56f133347f57acd7f09bb34f8/src/connectors/infinite-hits/__tests__/connectInfiniteHits-test.ts#L462-L479

Reply:
Got it, thanks! Let me check if I can avoid the ts ignore.